PhysicsTools/PatAlgos/python/slimming/lowPtGsfLinks_cfi.py

```diff
@@ -0,0 +1,10 @@
+import FWCore.ParameterSet.Config as cms
+
+lowPtGsfLinks = cms.EDProducer( 'LowPtGSFToPackedCandidateLinker',
```

Comment:
Now this file is a duplicate of the configuration produced during build by the fillDescription().
To avoid such a duplication you can either:
- Remove this file, and link directly ```PhysicsTools.PatAlgos.lowPtGsfLinks_cfi``` when needed. This is ok if you consider these parameters as reasonably fixed, and you don't expect that they can be modified at run time
- Rename this file (e.g.) ```lowPtGsfLinks_cff.py```, and make it import ```PhysicsTools.PatAlgos.lowPtGsfLinks_cfi```; then you can import ```PhysicsTools.PatAlgos.slimming.lowPtGsfLinks_cff.py``` where needed
- Rename the file automatically generated by fillDescriptions() by putting (e.g.) 
```C++
descriptions.add("lowPtGsfLinksDefault", desc);
```
then you can import here ```PhysicsTools.PatAlgos.lowPtGsfLinksDefault_cfi``` and clone the module ```lowPtGsfLinks``` from ```lowPtGsfLinksDefault```